src/state/user/reducer.ts

```diff
@@ -50,6 +51,9 @@ export interface UserState {
   showSurveyPopup: boolean | undefined
 
   showDonationLink: boolean
+
+  walletOverride: Wallet | null
+  walletOverrideBackfilled: boolean
```

Comment:
Is this meant to store whether the walletOverride has been set or not?
If so, can you instead set `walletOverride: Wallet | null | undefined`? It's unclear to me what `Backfilled` specifically does.

Reply:
i can change walletOverride to `Wallet | undefined`, it doesn't need to be null